web/pgadmin/misc/properties/ObjectNodeProperties.jsx

```diff
@@ -196,15 +196,22 @@ export default function ObjectNodeProperties({panelId, node, treeNodeInfo, nodeD
     && formType !== 'tab') {
     schema.filterGroups = [gettext('Security')];
   }
+  // To check node data is updated or not
+  const staleCounter = useRef(0);
 
   const key = useMemo(()=>{
```

Comment:
lets add one `useEffect` with `nodeData?._id` as dependency and set `staleCounter.current = 0` in that just to be safe.